### ISSUE TITLE
Move to ES6 Imports

### ISSUE THREAD

**Issue by absentabyss** [TeamPagnus/minecraft-server-manager#33]
Importing scripts in order on `index.html` is pretty messy. I think it would be better to use [ES6 imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).

**Comment by Sternenwarte88** [comment]
Hey, saw this as good first issue. I taken a look on using this problem and on your HTML file. It's right, that there is no css styling and all these? It's just a pure HTML site? 

I asked because, what's about making a react site of it?

**Comment by sridhar-5** [comment]
Hey, Just saw this issue. Would love to contribute but I'm trying to understand what the issue actually is. so are we required to replace the script tags used to make use of  javascript files with the ES6 import statements? @absentabyss. if I understood this right I don't think we can use ES6 imports in html file and if you really wanted to do that, as @Sternenwarte88 we can make a react site of it I think.

**Comment by absentabyss** [comment]
@Sternenwarte88, yeah it's just a plain HTML file just to get the project working. It would be great to improve the aesthetic with React and Tailwind.

**Comment by absentabyss** [comment]
@sridhar-5, script tags can be nasty and hard to follow, so I thought a good first issue would be to link all JS scripts to a main.js via ES6 imports, and then import only that main.js to the HTML. But yeah, a React site would be much better. :D

**Comment by sridhar-5** [comment]
Ohh..! I get you. If that's the thing to be done I can take this issue and open a pr by today. Can you assign me to this issue please?

**Comment by Sternenwarte88** [comment]
That's should be quite easy. Need some design sketches or work before that in figma? So we can first look how it's should look like and if that's UI/UX friendly.

@sridhar-5 or did you just want to start?

**Comment by absentabyss** [comment]
We can discuss the design of the front end at #34. I'll assign this current issue to @sridhar-5.

**Comment by Sternenwarte88** [comment]
That's okay. 
Saw in Javafiles often var. Maybe we should make in consense with ES6 and use let and const instead.

**Comment by sridhar-5** [comment]
Thanks I'll currently change the scripts to es6 imports and then we'll discuss about the react front end later sometime. opening PR now. @absentabyss 

Thanks..!

**Comment by sridhar-5** [comment]
Hey @absentabyss.
I tried to replace the scripts with imports but the main thing here is that if at all we have to use import statements we have to export all the functions in all the modules first. now since we have decide to switch to react this code will make no use. so it is better to archive this repository and we continue to work on react frontend,.

**Comment by absentabyss** [comment]
Good point. I don't think it's necessary to archive it. There's no use for the plain HTML Front End. It's currently just a placeholder.

I'll close this issue once a replacement front end is pulled.
